### PR TITLE
ENG-938: AdminConsole: amend menu (except CMS)

### PR DESCRIPTION
**Pull request by gReis89**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **brenoqvds**

If they are not already assigned, you can assign the PR to them by writing `/assign @brenoqvds` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-admin-console).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-admin-console/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/main/java/com/agiletec/apsadmin/global-messages_it.properties [hunk 1]
```diff
@@ -544,27 +544,27 @@ dashboard.status.work = Lavoro
...
 
-menu.pageDesigner.pageTree=Pagine
+menu.pageDesigner.pageTree=Gestione
 menu.pageDesigner.pageConfiguration=Designer
 menu.pageDesigner.pageSettings=Impostazione
 
-menu.UXPattern.widget=Micro frontend / Widget
-menu.UXPattern.fragments=UX Fragment
-menu.UXPattern.pageModels=Template di Pagina
+menu.UXPattern.widget=Micro frontends & Widgets
```
Comment (gidesan) [inline src/main/java/com/agiletec/apsadmin/global-messages_it.properties:565]:
```suggestion
menu.UXPattern.widget=Micro frontend & Widget
```

File: src/main/java/com/agiletec/apsadmin/global-messages_it.properties [hunk 2]
```diff
@@ -544,27 +544,27 @@ dashboard.status.work = Lavoro
...
+menu.UXPattern.fragments=UX Fragments
```
Comment (gidesan) [inline src/main/java/com/agiletec/apsadmin/global-messages_it.properties:566]:
```suggestion
menu.UXPattern.fragments=UX Fragment
```

File: src/main/java/com/agiletec/apsadmin/global-messages_it.properties [hunk 3]
```diff
@@ -585,11 +585,12 @@ menu.APPS.CMS.contentTypes=Tipi di Contenuto
 menu.APPS.CMS.contentModels=Modelli di Contenuto
 menu.APPS.CMS.contentSettings=Impostazione Contenuti
 
-menu.ECR=ECR
+menu.ECR=Repository
 
 menu.settings.categories=Categorie
-menu.settings.labelsLanguages=Etichette & lingue
+menu.settings.labelsLanguages=lingue & Etichette
```
Comment (gidesan) [inline src/main/java/com/agiletec/apsadmin/global-messages_it.properties:591]:
```suggestion
menu.settings.labelsLanguages=Lingue ed Etichette
```

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/entando-admin-console/pull/39#pullrequestreview-457512985" title="Approved">gidesan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-admin-console).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/entando-admin-console/blob/master/OWNERS)~~ [gidesan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**